### PR TITLE
Upgrade to go v1.13.4

### PR DESCRIPTION
**Pull request by chenrui333**
Upgrade to go v1.13.3

Relates to https://github.com/Homebrew/homebrew-core/pull/45962

### CODE REVIEW

**Comment by chenrui333** [comment]
test-coverage step failed, https://github.com/chenrui333/cli53/runs/280876291
```
go: github.com/chenrui333/cli53/internal/features: github.com/chenrui333/cli53@v0.0.0-20191029072227-b7516ac135f9: parsing go.mod:
	module declares its path as: github.com/barnybug/cli53
	        but was required as: github.com/chenrui333/cli53
##[error]Makefile:30: recipe for target 'test-coverage' failed
```

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -9,17 +9,15 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
-    - name: Set up Go 1.12
+    - name: Set up Go 1.13.3
```
Comment (barnybug) [inline .github/workflows/release.yml]: Latest stable is 1.13.4

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -7,10 +7,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
-    - name: Set up Go 1.12
+    - name: Set up Go 13.3
```
Comment (barnybug) [inline .github/workflows/test.yml]: `1.`...

File: .github/workflows/release.yml (hunk 1 above)
Comment (chenrui333) [reply .github/workflows/release.yml]: That sounds right, I am actually fully aware of this as homebrew-core maintainer 😄

File: .github/workflows/test.yml (hunk 2 above)
Comment (chenrui333) [reply .github/workflows/test.yml]: good catch, fixed it.

**Comment by chenrui333** [comment]
@barnybug Updated the PR to use the latest golang version. Let me know if it works. Thanks!

**Pull request merged**